server/config.yml

```diff
@@ -1,3 +1,6 @@
+log:
+  level: "INFO"
+  path: "log/logdetective.log"
```

Comment:
I'd argue to default to `/dev/stderr` given we are in the container world

What does everyone think?

Reply:
It's actually already logging to stdout and the logfile. See [utils.py](https://github.com/fedora-copr/logdetective/blob/043e108430a68ad82d558d0fde3801c24d3a0f7f/logdetective/server/utils.py#L29).

Reply:
oh, so it's extra; got confused, thanks for explanation!